### DISCUSSION TITLE
SendAlexaOfferToMaster Returns Null SDP Answer

### DISCUSSION THREAD

**Discussion by PPjamies** [aws/aws-sdk-java#2753] [Q&A] [answered]

**Comment by debora-ito** [comment, marked as answer]
@PPjamies Alexa Skills kit are maintained by the Alexa team, you can reach out to them in the https://github.com/alexa org, find the repo corresponding to the Alexa tool you're using.

This repo is dedicated to questions about AWS SDK for Java that communicates with the AWS Service APIs.
